lib/util/__fixtures__/release-notes.txt

```diff
@@ -0,0 +1,15 @@
+#### Our Gold Sponsors
+
+<table>
+</table>
+
+#### Our Silver Sponsors
+
+<table>
+</table>
+
+#### What's Changed
+* pnpm rebuild accepts --store-dir by @&#8203;user in https://github.com/foo/foo/pull/1
+
+#### New Contributors
+* @&#8203;user made their first contribution in https://github.com/foo/foo/pull/2
```

Comment:
Is there a reason why y0ou're using `@&#8203;`? It's a trick we use, but I've never seen it in raw release notes

Reply:
I ran Renovate on the reproduction repository and this is part of the returned value from `sanitizeMarkdown` function

![image](https://user-images.githubusercontent.com/98313426/187616911-66607d00-9fa2-4874-b44e-9b646b884c96.png)